app/src/main/java/org/p2p/wallet/settings/model/SettingsItemMapper.kt

```diff
@@ -53,10 +54,16 @@ class SettingsItemMapper(
     }
 
     private fun securityBlock(
+        isDeviceShareSaved: Boolean,
         isBiometricLoginEnabled: Boolean,
-        isBiometricLoginAvailable: Boolean
+        isBiometricLoginAvailable: Boolean,
     ): List<SettingsItem> = listOfNotNull(
         SettingsGroupTitleItem(groupTitleRes = R.string.settings_item_group_title_security),
+        ComplexSettingsItem(
+            nameRes = R.string.settings_item_title_recovery_kit,
+            iconRes = R.drawable.ic_settings_shield,
+            hasSeparator = true
+        ).takeIf { isDeviceShareSaved },
```

Comment:
only if device share is saved? 
what if we restore using social+custom? i think we need to show it too

Reply:
will fix it after API call impl